src/app/home-page/widget/organization-box/organization-box.component.html

```diff
@@ -52,6 +51,7 @@
           />
         </mat-form-field>
         <h5 class="weight-bold mt-0">My Organizations</h5>
+        <div *ngIf="listOfOrganizations.length === 0" class="text-align-center date-display mt-5">No matching search results.</div>
```

Comment:
I think his is going to "flash"; where when the page first loads, it will display this message, the request will complete, then this will go away. It would be nice if it didn't.

Reply:
Adding `&& !isLoading` to the `*ngIf` which avoids the problem, good detail thanks. Making a note to update the other `entry-boxes`